version/version.go

```diff
@@ -1,5 +1,28 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("version")
+
 var (
-	Version = "0.0.1"
+	Version   = "" // version will be replaced while building the binary using ldflags
+	GoVersion = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
I had asked for this to be removed during the previous review. Any reason to retain this? I don't see any reason to have an exported global variable that is not used anywhere.

Reply:
We are using `GoVersion` in `main.go` to print the go version.

Reply:
Then you typically add another API for that but I will let that go for now.

Reply:
And once you switch to using `version.Get()` in `main.go` you can remove this global variable.